### ISSUE TITLE
Circle buttons have artefacts in the pressed state

### ISSUE THREAD

**Issue by whitetown** [hirokimu/EMTNeumorphicView#5]
Round buttons have artifacts in the pressed state.
Steps to reproduce: 
 button0.cornerRadius = height/2

<img width="387" alt="unpressed" src="https://user-images.githubusercontent.com/10551374/81670115-6f5a5000-9447-11ea-9f49-c6f335d6ae28.png">

<img width="454" alt="pressed" src="https://user-images.githubusercontent.com/10551374/81669858-1985a800-9447-11ea-8886-2338a8b4db27.png">

**Comment by 11Shraddha** [comment]
Yes. i am also facing same issue for circle button. @whitetown Have you found any solution?

**Comment by tnducrocq** [comment]
I have the same problem... 
Solved with: button0.cornerRadius = height/2 - 1
There is no visible difference.
